Floofbot/Modules/Fun.cs

```diff
@@ -170,6 +172,23 @@ public async Task RequestBirb()
             }
         }
 
+        [Command("choice")]
+        [Summary("Chooses an option")]
+        public async Task Choice([Summary("Choices, delimited by ;")][Remainder]string choices = "")
+        {
+            if (!string.IsNullOrEmpty(choices))
+            {
+                string[] splitChoices = choices.Split(";", StringSplitOptions.RemoveEmptyEntries)
+                    .Select(choice => choice.Trim()).ToArray();
+                if (splitChoices.Length != 0)
+                {
+                    await Context.Channel.SendMessageAsync(splitChoices[rand.Next(splitChoices.Length)]);
+                    return;
+                }
+            }
+            await Context.Channel.SendMessageAsync("Not enough options were provided.");
```

Comment:
I would send something explaining how to use the delimiter, maybe provide an example